frontend/src/components/Pages/RobotPage/RobotPage.tsx

```diff
@@ -95,7 +95,7 @@ export const RobotPage = () => {
                                             batteryLevel={selectedRobot.batteryLevel}
                                             batteryWarningLimit={selectedRobot.model.batteryWarningThreshold}
                                         />
-                                        {selectedRobot.pressureLevel && (
+                                        {selectedRobot.pressureLevel != null && (
```

Comment:
Should we check for both null and undefined?

Reply:
I've added it, thanks for the catch!